graph.config.yml

```diff
@@ -48,6 +48,8 @@ contracts:
       reserveRatio: 500000 # 50% (parts per million)
       curationTaxPercentage: 25000 # 2.5% (parts per million)
       minimumCurationDeposit: "1000000000000000000" # 1 GRT
+      initializationPeriod: 432000 # Seconds = 5 Days
+      initializationExitPeriod: 2592000 # Seconds = 30 Days
```

Comment:
These will need to be adjusted once we finalize the GIP. They will also be affected by the upcoming Capital Gains Tax GIP as they both work in harmony to balance out incentives/disincentives.